Good-Bids/goodbids
# [#94] Metrics

# Summary

- Adds driver and metrics component
- Adjusts Tailwind content paths
- Adds inputs for new components
- Adds script to handle components in dev mode

## Issues

- #94

## Testing Instructions

- Start WP (`bin/start`) and Vite (`bin/start-vite`)
- Create an auction
- Visit said auction
- See the metrics block

## Screenshots

![image](https://github.com/Good-Bids/goodbids/assets/48295174/59a78e1b-54d2-4ad8-ad38-74183ec53e92)
